components/xsd-fu/templates-cpp/OMEXMLModelObject.template

```diff
@@ -994,7 +1071,7 @@ ${customUpdatePropertyContent[prop.name]}
       }
 {% end source %}\
 {% end %}\
-{% when prop.maxOccurs > 1 and not klass.isAbstract and not klass.isAbstractProprietary %}
+{% when prop.maxOccurs > 1 and not klass.isAbstract %}\
```

Comment:
In the cpp generated model code there are a number of places where new lines have disappeared. 
I cant see anywhere particular in the template that removes them but perhaps adding the forward slash (correctly) onto these template conditionals means that the set functions above for example no longer have any spacing between them.